main/src/ca/uwaterloo/flix/language/phase/jvm/BackendObjType.scala

```diff
@@ -855,6 +856,21 @@ object BackendObjType {
           LRETURN()
       ))
 
+    /*
+    def decAndCheck: StaticMethod = StaticMethod(this.jvmName, IsPublic, IsFinal, "decAndCheck",
+      mkDescriptor()(BackendType.Int64), Some(_ =>
```

Comment:
Returnerer -1 (for at det er nemt at kalde i TypedAst) og har fjernet ICONST_0 (EQ betyder == 0 i sig selv, lidt confusing)
```scala
mkDescriptor()(BackendType.Int32), Some(_ =>
  GETSTATIC(StepCounterField) ~
    INVOKEVIRTUAL(JvmName.AtomicLong, "getAndDecrement",
      MethodDescriptor(Nil, BackendType.Int64)) ~
    ifCondition(Condition.EQ) {
      thisLoad() ~ ACONST_NULL() ~
        ATHROW()
    } ~
    ICONST_M1() ~ LRETURN()
```

Reply:
#7426